pkg/scheduler/framework/plugins/noderesources/resource_allocation_test.go

```diff
@@ -552,18 +552,11 @@ func TestCalculateResourceAllocatableRequest(t *testing.T) {
 	}
 
 	for name, tc := range tests {
-		t.Run(name, func(t *testing.T) {
+		tCtx.SyncTest(name, func(tCtx ktesting.TContext) {
 			// Setup environment, create required objects
-			logger, tCtx := ktesting.NewTestContext(t)
-			tCtx, cancel := context.WithCancel(tCtx)
```

Comment:
This usage of `tCtx` for a `context.Context` was a bit unusual. Usually the name `tCtx` is used for a `test/utils/ktesting.TContext`, which is more than just a context (also includes the `testing.T` instances, for example).